### ISSUE TITLE
Site Title: Add a link to the block description instructions to go to the described location.

### ISSUE THREAD

**Issue by priethor** [WordPress/gutenberg#30871]
## What problem does this address?
The description in the Site Title Block mentions the Site Title can also be edited in "Settings > General", but the user needs to manually go to Settings, then to General, to see it.

![image](https://user-images.githubusercontent.com/27339341/114861750-f9483100-9ded-11eb-97a0-5d132cbea962.png)

## What is your proposed solution?

Link the "Settings > General" text to the actual destination screen.

**Comment by JustinyAhin** [comment]
Are there any examples of where such a thing is done?

**Comment by priethor** [comment]
Not that I can think of off the top of my head, since [the description](https://github.com/WordPress/gutenberg/blob/5a06916021e12b7cf04133c3b33aeffb0b618110/packages/block-library/src/site-title/index.js#L19) is a translatable string.

**Comment by ntsekouras** [comment]
Maybe an approach like this one: https://github.com/WordPress/gutenberg/pull/27732, with `createInterpolateElement`?

**Comment by Mamaduka** [comment]
This is no longer applicable after #34474. Closing as "won't fix".
